packages/vaadin-login/src/vaadin-login-mixin.js

```diff
@@ -70,6 +70,15 @@ export const LoginMixin = (superClass) =>
           notify: true
         },
 
+        /**
+         * If set, the user name field automatically receives focus when the component is attached to the document.
+         * @type {boolean}
```

Comment:
Please add the attribute annotation also here for consistency with the `d.ts` file:

```
@attr {boolean} enter-next-row
```

Reply:
Done